mesonbuild/backend/vs2010backend.py

```diff
@@ -356,6 +356,7 @@ def get_target_deps(self, t: T.Dict[T.Any, T.Union[build.Target, build.CustomTar
                     if isinstance(ldep, build.CustomTargetIndex):
                         all_deps[ldep.get_id()] = ldep.target
                     elif isinstance(ldep, File):
+                        # XXX: this this true? What if the file `is_built`?
```

Comment:
Is there an action to take here?

Reply:
I mean, probably. But I don't have a windows machine handy to find out what happens if a generated file is used for a link_depends and fix it.

Reply:
Ok. I will try to look at this (maybe not in a near future, but someday...)

Reply:
I did reduce it from an `XXX comment to just a comment. along with fixing the `this this` :)